client/src/app/Constants.ts

```diff
@@ -20,10 +20,13 @@ export interface IEnvVars {
   KEYCLOAK_CLIENT_ID: string;
   KEYCLOAK_SERVER_URL: string;
   UI_INGRESS_PROXY_BODY_SIZE: string;
+  PROFILE: string;
 }
 export const ENV: IEnvVars = (window as { [key: string]: any })["_env"] || {};
 export const isAuthRequired = ENV.AUTH_REQUIRED !== "false";
 export const uploadLimit = ENV.UI_INGRESS_PROXY_BODY_SIZE || "500m";
+export const isPropietaryAllowed =
```

Comment:
Can we think of a better name for this one since it is likely to be re-used for the branding also?

Reply:
In MTC we set the env var like so:

```
export enum BrandType {
  Crane = 'Crane',
  RedHat = 'RedHat',
}

export const APP_BRAND: BrandType = (process.env.BRAND_TYPE as BrandType) || BrandType.Crane;
```

So that we could render the brand image like:

```
 <img
                src={APP_BRAND === BrandType.RedHat ? logoRedHat : logoCrane}
                alt="Logo"
                className={
                  APP_BRAND === BrandType.RedHat ? styles.redhatLogoStyle : styles.craneLogoStyle
                }
              />

``` 

Not saying we should do this exactly, but just suggesting something slightly clearer.